systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterIT.java

```diff
@@ -283,36 +253,38 @@ public void testForUpdateValuesInConfigMap() {
             kubeClient.waitForResourceUpdate("pod", kafkaPodName(clusterName, i), kafkaPodStartTime.get(i));
             kubeClient.waitForPod(kafkaPodName(clusterName,  i));
         }
-        String configMap = kubeClient.get("cm", clusterName);
-        assertThat(configMap, valueOfCmEquals("zookeeper-healthcheck-delay", "23"));
-        assertThat(configMap, valueOfCmEquals("zookeeper-healthcheck-timeout", "24"));
-        assertThat(configMap, valueOfCmEquals("kafka-healthcheck-delay", "23"));
-        assertThat(configMap, valueOfCmEquals("kafka-healthcheck-timeout", "20"));
-        assertThat(configMap, valueOfCmEquals("kafka-config", "{\"default.replication.factor\": 2,\"offsets.topic.replication.factor\": 2,\"transaction.state.log.replication.factor\": 2}"));
-
-        LOGGER.info("Verified CM and Testing kafka pods");
+
+        LOGGER.info("Verify values after update");
+        String configMapAfter = kubeClient.get("cm", clusterName);
+        assertThat(configMapAfter, valueOfCmEquals("zookeeper-healthcheck-delay", "31"));
```

Comment:
Why did you change the values?

Reply:
Honestly? I think the values should have some meaning where possible. If we think that timeout in the first place should be 10, why do we change it to something completely random such as `23`? And on top of it, if the timeout is actually used, it will take for the tests longer to succeed or fail.

Reply:
OIC, you just took the existing defaults and added one to them. Makes more sense now.

Reply:
Yeah, exactly, that was the idea.